schema.sql

```diff
@@ -25,3 +24,27 @@ CREATE TABLE species (
 	name VARCHAR(100) NOT NULL,
 	PRIMARY KEY (specie_id)
 );
+
+CREATE TABLE vets (
+    vet_id SERIAL,
+	name VARCHAR(100) NOT NULL,
+	age INT NOT NULL,
+	date_of_graduation date NOT NULL,
+	PRIMARY KEY (vet_id)
+);
+
+-- Create intersection tables 
+CREATE TABLE specializations(
+  id SERIAL,
+  specie_id INT REFERENCES species(specie_id),
+  vet_id INT REFERENCES vets(vet_id),
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE visits(
+  id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
- [  ] It would be better if we address the comment made by the previous code reviewer. You are suppose to implement this functionality
```
- Remove column species
- Add column species_id which is a foreign key referencing species table
- Add column owner_id which is a foreign key referencing the owners table
```
Inside the schema.sql file. If you are not clear with the comments, kindly reach out

Reply:
I have actually done that in the data.sql file lines 125 - (which is where we were asked to do so)
However, at your request, I'll include that in the schema.sql file.  

Thank you